src/ImmunizationHistory.js

```diff
@@ -60,6 +60,9 @@ function renderImmunizationGroup(
     }
   };
   const renderPerformers = (performers) => {
+
+	if (!performers) return(undefined);
```

Comment:
Oops forgot about this; it's just a defensive check I noticed when scanning my COVID card that didn't have an entry for performers.

Reply:
good catch!